experiments/experiments-api/src/main/java/com/duckduckgo/experiments/api/VariantManager.kt

```diff
@@ -34,8 +34,10 @@ interface VariantManager {
     fun updateAppReferrerVariant(variant: String)
 
     /**
-     * Persists experimental variants in the local table when remote config for variantManager
+     * Persists experimental variants in the local database when the remote configuration for variantManager
```

Comment:
Same in the documentation, the variant manager should not know about where the variants come from. We should not mention remote config here